develop-docs/sdk/expected-features/index.mdx

```diff
@@ -91,6 +91,27 @@ Local variable names and values for each stack frame, where possible. Restrictio
 
 This functionality should be gated behind the `includeLocalVariables` option, which is `true` by default.
 
+## Feature Flags
+
+An SDK may expose a Scope property for tracking feature flag evaluations. When the scope forks its important to clone the feature flags property. Leaking flag evaluations between threads could lead to inaccurate feature flag evaluation logs.
+
+The Scope's flag property should have a capped capacity and should prefer recently-evaluated flags over less-recently-evaluated flags. The recommended data structure is a LRU-cache but it is not required so long as the data structure behaves similarly. Serious deviations from the behavior of an LRU-cache should be documented for your language.
+
+The Scope's flag property should expose two methods: `get/0` and `set/2`. `set/2` takes two arguments. The first argument is the name of the flag. It is of type string. The second argument is the evaluation result. It is of type boolean. `set/2` should remove all entries from the LRU-cache which match the provided flag's name and append the new evaluation result to the end of the queue. `get/0` accepts zero arguments. The `get/0` method must return a list of serialized flag evaluation results in order of evaluation. Oldest values first, newest values last. See the <Link to="/sdk/data-model/event-payload/contexts/#feature-flag-context">Feature Flag Context</Link> protocol documentation for details.
```

Comment:
Should we mention the differences with the JS implementation in this section?

Reply:
This guide is to specifically guide new SDKs.  Since we've already written the JS SDK with its exceptions I don't think its super critical.  But we could.  Maybe in a Javascript specific page.